src/AspNetCore.Lambda/HttpHandlers.fs

```diff
@@ -275,4 +277,15 @@ let htmlFile (relativeFilePath : string) =
                 ctx
                 |> (setHttpHeader "Content-Type" "text/html"
                 >>= setBodyAsString html)
-        }
\ No newline at end of file
+        }
+
+let razor (engine : RazorLightEngine) (view : string) (model : obj) =
+    let view = engine.Parse(view, model)
+    setHttpHeader "Content-Type" "text/html"
+    >>= setBodyAsString view
+
+let configureRazor (viewsFolder : string)  = 
+    let views = combinePaths (Directory.GetCurrentDirectory()) viewsFolder
+    razor (EngineFactory.CreatePhysical(views))
+
+let defaultRazor () = configureRazor ""
```

Comment:
I was thinking that instead of having to create the `RazorLightEngine` every time you could merge all 3 functions to one like this perhaps:

```
let razorView (viewName : string) (model : obj) =
    fun (ctx : HttpHandlerContext) ->
        let engine = ctx.Services.GetService<IRazorLightEngine>()
        let view = engine.Parse(viewName, model)
        setHttpHeader "Content-Type" "text/html"
        >>= setBodyAsString view
        <| ctx
```

Here the HttpHandler tries to resolve an `IRazorLightEngine` from the service provider.

You could create an extension method like this:

```
type IServiceCollection with
    member this.AddRazorEngine (viewsFolderPath : string) =
        this.AddSingleton<IRazorLightEngine>(EngineFactory.CreatePhysical(viewsFolderPath));
```

... which then can be used like this:

```
        services.AddRazorEngine("C:\\AspNetCore.Lambda.SampleApp\\views") |> ignore
```

I think this would be more incline with existing ASP.NET Core patterns and it would allow users to configure the engine only once as a singleton or provide a custom implementation.

What do you think?